integration-tests/metrics_test.go

```diff
@@ -424,9 +423,13 @@ func checkMetrics(
 			require.NotContains(t, lines, fmt.Sprintf("%v", getPrometheusNameWithSuffix(prefix, metrics.OriginRequestDuration, "count")))
 		}
 
-		requireEventuallyContainsLine(t, lines, fmt.Sprintf("%v %v", getPrometheusName(prefix, metrics.AvailableStreamIdsOrigin), streamIdsOrigin))
-		requireEventuallyContainsLine(t, lines, fmt.Sprintf("%v %v", getPrometheusName(prefix, metrics.AvailableStreamIdsTarget), streamIdsTarget))
-		requireEventuallyContainsLine(t, lines, fmt.Sprintf("%v %v", getPrometheusName(prefix, metrics.AvailableStreamIdsAsync), streamIdsAsync))
+		if (successOrigin + successBoth) == 0 {
```

Comment:
why this if?

Reply:
I thought these would indicate successful request execution, but it looks like they're that and not needed. Removed.